ui/components/multichain/asset-picker-amount/asset-picker/asset-picker.tsx

```diff
@@ -127,12 +129,12 @@ export function AssetPicker({
       : symbol;
 
   // Badge details
-  const currentChainId = useSelector(getCurrentChainId);
-  const allNetworks = useSelector(getNetworkConfigurationsByChainId);
-  const currentNetwork = allNetworks[currentChainId];
-  const selectedNetwork =
-    networkProps?.network ??
-    (currentNetwork?.chainId && allNetworks[currentNetwork.chainId]);
+  const currentChainId = useMultichainSelector(getMultichainCurrentChainId);
+  const allNetworks = useSelector(getMultichainNetworkConfigurationsByChainId);
+  const currentNetwork_ =
```

Comment:
Can we find a more meaningful name than `currentNetwork_` alongside `currentNetwork`?